packages/core/src/codewhisperer/commands/basicCommands.ts

```diff
@@ -53,16 +56,31 @@ export const toggleCodeSuggestions = Commands.declare(
             })
 
             const isSuggestionsEnabled = await suggestionState.toggleSuggestions()
+
             span.record({
                 settingState: isSuggestionsEnabled
                     ? CodeWhispererConstants.autoSuggestionConfig.activated
                     : CodeWhispererConstants.autoSuggestionConfig.deactivated,
             })
             vsCodeState.isFreeTierLimitReached = false
+
+            isSuggestionsEnabled
+                ? void showTimedMessage('Amazon Q: Auto-Suggestions are currently running.', MessageTimeOut)
+                : void showTimedMessage('Amazon Q: Auto-Suggestions are currently paused.', MessageTimeOut)
```

Comment:
using a ternary may be clearer and less verbose

Reply:
this kind of message is probably more appripriate as a status message rather than a popup